### ISSUE TITLE
[Offline docs] Links on the Search Results Page don't work. They are blocked by CORS & Not allowed to load local resource

### ISSUE THREAD

**Issue by chrisfcarroll** [fish-shell/fish-shell#8019]
```
fish --version= 3.2.2
Darwin 20.4.0 Darwin Kernel Version 20.4.0: Thu Apr 22 21:46:47 PDT 2021; root:xnu-7195.101.2~1/RELEASE_X86_64 x86_64
xterm-256color
Browser: Safari but also Edge etc.
```
XMLHttpRequests on localhost are pain for webdevelopers because since a couple of years ago the leading browsers all moved to enforcing CORS on localhost and refusing XMLHttpRequests access to  resources from a `file://` url.

fish help for 3.2.2 mostly works, except **the links on the search page after a search**. They result in this kind of message hidden in the browser Console:

```
[Error] Cross origin requests are only supported for HTTP.
	send (jquery.js:9599)
	ajax (jquery.js:9205)
	loadIndex (searchtools.js:87)
	(anonymous function) (search.html:22)
	mightThrow (jquery.js:3535)
	(anonymous function) (jquery.js:3603)
[Error] XMLHttpRequest cannot load file:///usr/local/Cellar/fish/3.2.2_1/share/doc/fish/searchindex.js due to access control checks.
	send (jquery.js:9599)
	ajax (jquery.js:9205)
	loadIndex (searchtools.js:87)
	(anonymous function) (search.html:22)
	mightThrow (jquery.js:3535)
	(anonymous function) (jquery.js:3603)

Not allowed to load local resource: file:///usr/local/Cellar/fish/3.2.2_1/share/doc/fish/cmds/disownundefined?highlight=lists
```

which might mostly simply be fixed by making them plain links instead of javascripty links?

**Comment by faho** [comment]
Duplicate of #7946

**Comment by faho** [comment]
The error message is misleading - the links are broken because your docs were generated by an older version of sphinx that didn't have one variable defined. We've since worked around that.

The links *are* ordinary `<a href>` style links, it's just that they're pointing to the wrong place in your version - `disownundefined` should be `disown.html`.

**Comment by faho** [comment]
Also: If this truly didn't work you'd report it to upstream sphinx, not us, because then it would be a general problem for them.

**Comment by chrisfcarroll** [comment]
👍 Noted, ta. Sorry I didn't manage to find #7946 before creating this issue.
